lib/libc/minimal/source/stdlib/rand.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Space Cubics, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <toolchain.h>
```

Comment:
Is this include really needed?

Reply:
nop.  I'll leave `stdlib.h` there for checking the `rand()` and `srand()` declarations against the definitions.